WcaOnRails/config/routes.rb

```diff
@@ -321,6 +321,14 @@
         patch '/wcif' => 'competitions#update_wcif', as: :update_wcif
       end
       get '/records' => "api#records"
+
+      scope '/roles' do
+        get '/' => 'roles#get', as: :roles_get
+        post '/' => 'roles#post', as: :roles_post
+        patch '/' => 'roles#patch', as: :roles_patch
+        delete '/' => 'roles#delete', as: :roles_delete
+        get '/list' => 'roles#list', as: :roles_list
+      end
```

Comment:
I don't mean using `scope`. I specifically meant using `resources`. Please refer to https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions again.

I am fairly certain that this whole section can be replaced with one single line:

```ruby
resources :roles
```

Reply:
Oh okay, I was trying something else that's why it didn't work. Now changed.